firmware/quadruna/VC/src/app/states/app_allStates.c

```diff
@@ -24,6 +24,7 @@ static uint16_t heartbeat_cycles = 0;
 void app_allStates_runOnTick100Hz(void)
 {
     // Enable PCM if HV up.
+    // TODO: move this to PCM state?
```

Comment:
I agree this should be moved to PCM state. I am not sure why we decided to run it in app_allStates_runOnTick100Hz